package.json

```diff
@@ -34,15 +34,16 @@
     "prepublishOnly": "yarn run build"
   },
   "dependencies": {
-    "axios": "~1.6.5",
     "pluralize": "8.0.0"
   },
   "devDependencies": {
+    "@peculiar/webcrypto": "^1.4.5",
     "@types/jest": "29.5.3",
     "@types/node": "14.18.54",
     "@types/pluralize": "0.0.30",
     "axios-mock-adapter": "1.21.5",
```

Comment:
Do we still need this?

Reply:
Yes, see https://github.com/workos/workos-node/pull/962#discussion_r1484519580

Reply:
I was referring to `"axios-mock-adapter": "1.21.5",`, looks like it's not used anywhere.

Reply:
oh I see!